package.json

```diff
@@ -85,6 +85,7 @@
       "<rootDir>/src/"
     ],
     "unmockedModulePathPatterns": [
+      "<rootDir>/src/tools/RelayUnstableBatchedUpdates.js",
```

Comment:
Honestly, I'm not sure how to do that :( Sorry...not a Jest expert. If you point me in the right direction, happy to make the change...otherwise I defer to your decision on that.

Based on my limited knowledge of requireActual...it seems like it would work fine. But I'm not 100% certain.